models/Airplane/Airplane.jl

```diff
@@ -14,7 +14,18 @@
 # where ``(x, y, z)`` is the position of the C.G., ``(u, v, w)`` are the
 # components of velocity in ``(x, y, z)`` directions, ``(p, q, r)`` are body
 # rotation rates, and ``(ϕ, θ, ψ)`` are the Euler angles.
-#
+
```

Comment:
Above this line "is the position of the C.G." -> "is the position of the center of gravity (C.G.)"